config.txt

```diff
@@ -7,5 +7,4 @@
 -as https://openbis1605test.am10.uni-tuebingen.de/openbis/openbis
 
 # Set the DSS_URL (DataStoreServerURL) to the value defined below
-# -dss https://qbis.qbic.uni-tuebingen.de:444/datastore_server
--dss https://openbis1605test.am10.uni-tuebingen.de/datastore_server
+-dss https://qbis.qbic.uni-tuebingen.de:444/datastore_server,https://qbis.qbic.uni-tuebingen.de:444/datastore_server2
```

Comment:
really the same port for both, @sven1103 ?

Reply:
updated 😄